index.html

```diff
@@ -139,7 +139,75 @@ <h2>What is a Hackathon?</h2>
     </section>
     <section id="schedule">
     </section>
-    <section id="faq">
+    <section id="faq-content">
+        <div id="faq-wrapper">
+            <h1>Frequently Asked Questions</h1>
+            <div id="faq-cards">
+                <div class="card">
+                    <button class="accordion"><i class="fa fa-plus fa-plus-pink" aria-hidden="true"></i>Who can participate in BrickHack?</button>
+                    <div class="panel">
+                        <p>
```

Comment:
More concise to put the text on the same line as the `<p>` and have the closing tag also be inline w/ the text:
```html
<p>asdfasdfadsfasdf...asdfasdfasdf</p>
```